python-frontend/src/main/java/org/sonar/python/cfg/ControlFlowGraphBuilder.java

```diff
@@ -67,13 +70,41 @@ public ControlFlowGraphBuilder(@Nullable StatementList statementList) {
     exitTargets.push(end);
     if (statementList != null) {
       start = build(statementList.statements(), createSimpleBlock(end));
+      addParametersToStartBlock(statementList);
```

Comment:
Good idea :bulb:. We should merge this PR before SONARPY-481, so I can change the workaround in my PR to look for the parameters once I reached the `start` block.